wesnoth-ei.po

```diff
@@ -323,22 +325,22 @@ msgstr ""
 #. [achievement_group]
 #: data/campaigns/Eastern_Invasion/achievements.cfg:71
 msgid "Kill no enemies before defeating Mal-Ravanal in ‘<i>The Duel</i>’."
-msgstr ""
+msgstr "Nezabij žádného nepřítele, než v <i>Duelu</i>porazíš Mal-Ravanala."
```

Comment:
Zachoval bych v překladu dvojité zvýraznění (uvozovky, kurzíva), přestože u úspěchů v jiných taženích je zvýraznění pouze kurzívou (hra více hráčů) nebo není krom velkého písmena žádné (Svoboda). Až se tam nahoře rozhodnou to sladit, my opravíme fuzzy, ale nezůstane nám zapomenutá jinakost, kterou jsme se snažili opravit chybu v předloze.
```suggestion
msgstr "Nezabij žádného nepřítele, než v „<i>Duelu</i>“ porazíš Mal-Ravanala."
```